.circleci/config.yml

```diff
@@ -1,26 +1,72 @@
 version: 2.1
 orbs:
-  buildpacks: jkutner/buildpacks@0.1.0
-  ruby: circleci/ruby@0.2.1
+  pack: buildpacks/pack@0.2.4
+
 jobs:
   build:
-    machine: true
+    machine:
+      image: ubuntu-2004:202111-02
     steps:
-      - buildpacks/install-pack:
-          version: "0.11.0"
       - checkout
+      - pack/install-pack:
+          version: 0.23.0
+      - restore_cache:
+          keys:
+            - v3-ruby-3.0.3
+            - v3-deps
```

Comment:
We can get rid of that then. The big thing I really want to avoid is having to re-compile Ruby from scratch. Avoiding the hatchet install and bundle install saves time, but it's trivial compared to compiling Ruby. I'll update and remove this to clear up confusion.